util/MeasurePopulations.ts

```diff
@@ -6,43 +6,72 @@ export interface MultiSelectData {
   disabled: boolean;
 }
 
+export const PopulationShorthand: Record<string, string> = {
+  'initial-population': 'IPP',
+  denominator: 'DENOM',
+  'denominator-exclusion': 'DENEX',
+  'denominator-exception': 'DENEXCEP',
+  numerator: 'NUMER',
+  'numerator-exclusion': 'NUMEX',
+  'measure-population': 'MSRPOPL',
+  'measure-population-exclusion': 'MSRPOPLEX',
+  'measure-observation': 'OBSERV'
+};
+
```

Comment:
Rather than adding this const, I think can use the PopulationType enum exported from fqm-execution. What are your thoughts on this?

Reply:
This might be a little tough to do in code and might even be impossible, I'd maybe just leave this as is

Reply:
I think this kind of reverse mapping to get the Enum name is possible for numeric enums (https://www.typescriptlang.org/docs/handbook/enums.html#reverse-mappings), but I'm not seeing an equivalent for string enums. @elsaperelli - okay with giving up on making this change?